packages/opentelemetry-node/test/NodeTracerProvider.test.ts

```diff
@@ -160,25 +165,29 @@ describe('NodeTracerProvider', () => {
         logger: new NoopLogger(),
       });
 
-      const sampledParent = provider
-        .getTracer('default')
-        .startSpan('not-sampled-span', {
-          parent: {
-            traceId: 'd4cda95b652f4a1592b449d5929fda1b',
-            spanId: '6e0c63257de34c92',
-            traceFlags: TraceFlags.NONE,
-          },
-        });
+      const sampledParent = provider.getTracer('default').startSpan(
+        'not-sampled-span',
+        {},
+        setExtractedSpanContext(ROOT_CONTEXT, {
```

Comment:
why the context needs to be wrapped here, instead of using just a context ?
```
{
  traceId: 'd4cda95b652f4a1592b449d5929fda1b',
  spanId: '6e0c63257de34c92',
  traceFlags: TraceFlags.NONE
}
```

Reply:
I don't understand. The third argument to `startSpan` is a `Context` not a `SpanContext`. Specifying parent as `SpanContext` directly is now forbidden by spec. This sets the parent on the context, then passes it into `startSpan`.